openfl/interface/workspace.py

```diff
@@ -124,19 +130,14 @@ def export_():
     from plan import freeze_plan
     from openfl.interface.cli_helper import WORKSPACE
 
-    # TODO: Does this need to freeze all plans?
     plan_file = Path('plan/plan.yaml').absolute()
     try:
         freeze_plan(plan_file)
     except Exception:
         echo(f'Plan file "{plan_file}" not found. No freeze performed.')
 
-    from pip._internal.operations import freeze
-    requirements_generator = freeze.freeze()
-    with open('./requirements.txt', 'w') as f:
-        for package in requirements_generator:
-            if is_package_versioned(package):
-                f.write(package + '\n')
+    # Dump requirements.txt
+    dump_requirements_file(prefixes=pip_install_options, keep_original_prefixes=True)
```

Comment:
dump_requirements_file reads requirements.txt file to get prefixes form it and then rewrite that file by merged prefixes (pip_install_options with prefixes from requirements.txt) and installed packages (from `pip freeze`). Probably we should explicitly use two different file, one as a source of prefixes  and write to other?

Reply:
we do not manage different versions of requirements files